api/v1_track_comment_count.go

```diff
@@ -36,6 +36,13 @@ func (app *ApiServer) v1TrackCommentCount(c *fiber.Ctx) error {
 		WHERE score < 0
 	),
 
+	-- Deactivated users
+	deactivated_users AS (
```

Comment:
Wouldn't this potentially be a huge set that we mostly don't use? Wondering if this is better as just a join on users and then filter afterwards on users.is_deactivated. Maybe I don't understand how the query planning works. Does it evaluate this whole result set and store it before moving on? Or would it pull from this query as needed for processing the final result set?

Reply:
inlined by default!